Web/Models/Entities/Club.php

```diff
@@ -281,14 +290,24 @@ function getManagers(int $page = 1): \Traversable
         }
     }
 
-    function getManager(User $user): ?Manager
+    function getManager(User $user, bool $ignoreHidden = false): ?Manager
     {
-        return (new Managers)->getByUserAndClub($user->getId(), $this->getId());
+        $manager = (new Managers)->getByUserAndClub($user->getId(), $this->getId());
+
+        if ($ignoreHidden && $manager !== null && $manager->isHidden()) {
+            $manager = null;
+        }
+
+        return $manager;
     }
     
-    function getManagersCount(): int
+    function getManagersCount(bool $ignoreHidden = false): int
     {
-        return sizeof($this->getRecord()->related("group_coadmins.club")) + 1;
+        if ($ignoreHidden) {
+            return sizeof($this->getRecord()->related("group_coadmins.club")->where("hidden", false)) + (int) !$this->isOwnerHidden();
```

Comment:
Is it nescessary to use an if/else here? I think it can be omitted:
```php
if(...)
    return ...;
    
return ...;```

Reply:
Done